lib/tasks/scripts/correct_bangladesh_medication_dosages.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+class CorrectBangladeshMedicationDosages
+  PROTOCOL_DOSAGES = {
+    'Amlodipine' => '5 mg',
+    'Losartan Potassium' => '50 mg',
+    'Hydrochlorothiazide' => '12.5 mg'
+  }.freeze
+
+  def self.call(*args)
+    new(*args).call
+  end
+
+  attr_reader :dryrun, :verbose
+
+  def initialize(dryrun: false, verbose: true)
+    @dryrun = dryrun
+    @verbose = verbose
```

Comment:
This is to prevent it from polluting the specs I assume? Nice.

Reply:
Correct, inspired by Rob's work in https://github.com/simpledotorg/simple-server/pull/963